resources/contenttweaker/blockstates/amuileria_kaerunea.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Does contenttweaker/blockstates files generated automatically? If I will not push them into modpack, will it hurt?

Reply:
These are generated automatically, you can not push them